com.gluonhq.SceneBuilder.yml

```diff
@@ -0,0 +1,60 @@
+app-id: com.gluonhq.SceneBuilder
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: SceneBuilder-runner
+rename-desktop-file: scenebuilder-SceneBuilder.desktop
+finish-args:
+  - --env=PATH=/app/bin
+  - --socket=x11
+  - --share=ipc
+  - --share=network
+  - --persist=.scenebuilder
+  - --filesystem=home
+cleanup:
+  - /share/info/*
+  - /share/locale/*
+  - /share/man/*
+  - /share/runtime/locale/*
+modules:
+  - name: SceneBuilder
+    buildsystem: simple
+    build-commands:
+    - mkdir -p deb-package
+    - ar p SceneBuilder.deb data.tar.xz | tar -xJf - -C deb-package
+    - mv deb-package/opt/scenebuilder /app
+    - install -dm755 /app/bin
+    - install -m755 SceneBuilder-runner /app/bin/SceneBuilder-runner
+    - install -Dm644 com.gluonhq.SceneBuilder.metainfo.xml -t /app/share/metainfo/
+    - install -Dm644 com.gluonhq.SceneBuilder.256.png /app/share/icons/hicolor/256x256/apps/com.gluonhq.SceneBuilder.png
+    - install -Dm644 com.gluonhq.SceneBuilder.128.png /app/share/icons/hicolor/128x128/apps/com.gluonhq.SceneBuilder.png
+    - desktop-file-edit --remove-key=Categories /app/scenebuilder/lib/scenebuilder-SceneBuilder.desktop
+    - desktop-file-edit --set-key=Categories --set-value='Development' /app/scenebuilder/lib/scenebuilder-SceneBuilder.desktop
```

Comment:
why no using `add-category` ?

Reply:
`Running: desktop-file-edit --set-icon=com.gluonhq.scenebuilder /app/scenebuilder/lib/scenebuilder-SceneBuilder.desktop
/app/scenebuilder/lib/scenebuilder-SceneBuilder.desktop: error: value ""Development;Building;GUIDesigner;Java;"" for key "Categories" in group "Desktop Entry" contains an unregistered value ""Development"; values extending the format should start with "X-"
/app/scenebuilder/lib/scenebuilder-SceneBuilder.desktop: error: value ""Development;Building;GUIDesigner;Java;"" for key "Categories" in group "Desktop Entry" contains an unregistered value """; values extending the format should start with "X-"
Error on file "/app/scenebuilder/lib/scenebuilder-SceneBuilder.desktop": Failed to validate the created desktop file
`

Reply:
original value seem to be causing issues